modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/release-notes/fragments/release-notes-tabs/index.html

```diff
@@ -6,12 +6,24 @@
 	[#assign currentTabId = (currentDisplayPaths?first)! /]
 
 	[#assign currentDisplayIds = (displayPortalURLs?last?split(currentTabId)?last)! /]
+
+	[#assign
+		paginationURLParameter = "&start="
+		paginationInitialValue = "1" /]
+
+	[#assign paginationParameterIndex = currentDisplayIds?index_of(paginationURLParameter) /]
+
+	[#if paginationParameterIndex != -1]
+		[#assign beforePaginationParameter = currentDisplayIds?substring(0, paginationParameterIndex + paginationURLParameter?length) /]
+
+		[#assign currentDisplayIdsParameterized = beforePaginationParameter + paginationInitialValue /]
```

Comment:
We shouldn't declare a variable in an if statement and call it outside, since there is the chance we call it and its unset